src/lib/postInventory.ts

```diff
@@ -0,0 +1,26 @@
+import { infoMessages } from "@/data/responseMessages";
+import { apiRoutes } from "@/data/ROUTES";
+import { Inventory } from "@/model/InventoryModel";
+
+
+export async function getInventoryByHistory(email: string, historyId: number, objectId: number): Promise<Inventory[]> { 
```

Comment:
if it is a method to GET inventory according to the story, keep it, else change method name to prcise it is a POST